### ISSUE TITLE
[enhancement] add rarity changes affected by looting in config

### ISSUE THREAD

**Issue by ajaxnotmeta** [onebeastchris/customplayerheads#39]
i know its weird to ask but may you add rarity changes affected by looting in config? i have 2 different server with different style, one of them were made to make Player head as a throphy of winning a "War" competition where the head must drop once it killed, another one were casual survival where the heads made to be RNG based where theres a changes when you kills someone you get its head. with no way to configure this i am hesitated to put this on both servers and adding bedrock player into the server

**Comment by onebeastchris** [comment]
Do you want to change the likelihood of the head being dropped upon being killed?
Not sure how you'd like rarity and the enchantments to interact. I'm open to suggestions though!

**Comment by ajaxnotmeta** [comment]
yeah theres a likelihood of the head being dropped upon being killed affected by the Looting enchantment, similar to how Wither Skeleton head behaves in vanilla, but with a config where we can turn this features on or off, such use case was that server where every kills always drop the head(no likelihood just drops 100%) and the other server where every kills has a likelihood it drops the head with Looting enchantment boost the likelihoodness of the head drops by certain percent
